gravitee-apim-console-webui/src/management/api/creation-v4/steps/step-2-entrypoints/step-2-entrypoints-0-architecture.component.html

```diff
@@ -81,6 +81,18 @@
             </gio-selection-list-option-layout-body>
           </gio-selection-list-option-layout>
         </div>
+        <div *ngIf="!hasLicense">
+          <gio-banner-error>
+            This configuration requires an enterprise license
+            <span gioBannerBody
+              >Request a license to unlock enterprise functionality, such as support for event brokers, asynchronous APIS, and Webhook
+              subscriptions.</span
+            >
+            <div gioBannerAction>
+              <button mat-raised-button color="basic" (click)="licenseService.displayUpgradeCta($event)">Request upgrade</button>
+            </div>
+          </gio-banner-error>
+        </div>
```

Comment:
```suggestion
<gio-banner type="error" icon="gio:universe">
            This configuration requires an enterprise license
            <span gioBannerBody
              >Request a license to unlock enterprise functionality, such as support for event brokers, asynchronous APIS, and Webhook
              subscriptions.</span
            >
            <div gioBannerAction>
              <button mat-raised-button color="basic" (click)="licenseService.displayUpgradeCta($event)">Request upgrade</button>
            </div>
          </gio-banner>
```

This way you will get the same icon as in the mockup